nuget/Xamarin.Controls.SignaturePad.nuspec

```diff
@@ -31,14 +34,23 @@
     <file src="output/android/SignaturePad.dll" target="lib/MonoAndroid/SignaturePad.dll" />
     <file src="output/android/SignaturePad.xml" target="lib/MonoAndroid/SignaturePad.xml" />
 
+    <file src="output/wpf/SignaturePad.dll" target="lib/net472/SignaturePad.dll" />
+    <file src="output/wpf/SignaturePad.xml" target="lib/net472/SignaturePad.xml" />
+
+    <file src="output/gtk/SignaturePad.dll" target="lib/net471/SignaturePad.dll" />
+    <file src="output/gtk/SignaturePad.xml" target="lib/net471/SignaturePad.xml" />
+
     <file src="output/ios/SignaturePad.dll" target="lib/Xamarin.iOS/SignaturePad.dll" />
     <file src="output/ios/SignaturePad.xml" target="lib/Xamarin.iOS/SignaturePad.xml" />
+	
+    <file src="output/mac/SignaturePad.dll" target="lib/Xamarin.Mac/SignaturePad.dll" />
+    <file src="output/mac/SignaturePad.xml" target="lib/Xamarin.Mac/SignaturePad.xml" />
 
     <file src="output/uwp/SignaturePad.dll" target="lib/uap10.0/SignaturePad.dll" />
     <file src="output/uwp/SignaturePad.xml" target="lib/uap10.0/SignaturePad.xml" />
     <file src="output/uwp/SignaturePad.pri" target="lib/uap10.0/SignaturePad.pri" />
-    <file src="output/uwp/Themes/Generic.xbf" target="lib/uap10.0/SignaturePad/Themes/Generic.xbf" />
-    <file src="output/uwp/SignaturePad.xr.xml" target="lib/uap10.0/SignaturePad/SignaturePad.xr.xml" />
+    <file src="output/uwp/SignaturePad/Themes/Generic.xaml" target="lib/uap10.0/SignaturePad/Themes/Generic.xaml" />
+    <file src="output/uwp/SignaturePad/SignaturePad.xr.xml" target="lib/uap10.0/SignaturePad/SignaturePad.xr.xml" />
```

Comment:
These two files dont seem to be ending up in the output directory

Reply:
build output for uwp
![image](https://user-images.githubusercontent.com/33021114/77953079-5cdfe900-72d5-11ea-8ff0-ff818bf207ce.png)

Reply:
I apologize, I was not clear, I see them there but I do not see them in the nuget package. I see it in the signaturePad nuget package but not the signaturepad.forms package
![image](https://user-images.githubusercontent.com/13969340/77966415-d78a1380-7297-11ea-96c0-00e19996f968.png)

![image](https://user-images.githubusercontent.com/13969340/77966430-dfe24e80-7297-11ea-9ecc-ace01b317645.png)


Makes me think its just not getting copied to one of the packages

Reply:
I would appreciate, if you create a PR, so I will check I setup permissions right.